ops/op2/test_cases/sync/doc_comment.out

```diff
@@ -28,7 +28,8 @@ impl op_has_doc_comment {
         let result = Self::call();
     }
     fn fast_function(_: deno_core::v8::Local<deno_core::v8::Object>) -> () {
-        Self::call()
+        let result = Self::call();
+        result
```

Comment:
Won't this bit raise clippy errors of unnecessary binding?

Reply:
It might, but these will pop up as we start porting ops. I can try one or two ops as part of this patch to flush them out.

Reply:
I decided to leave it for now -- as we port ops we can start to add the warning overrides as they are pretty simple to add.